lib/Internal/AutoDisposingPipeline.php

```diff
@@ -26,7 +26,7 @@ public function __construct(EmitSource $source)
 
     public function __destruct()
     {
-        $this->source->dispose();
+        $this->source->destroy();
```

Comment:
Why destroy instead of dispose?

Reply:
The generator can keep running even if the `AsyncGenerator` object was destroyed, so it can keep emitting values to pending promises from `continue()` calls before the instance was GC'd.